pumpio/pumpio.php

```diff
@@ -361,20 +362,17 @@ function pumpio_settings_post(App $a, array &$b)
 			$host = trim($host);
 			$host = str_replace(["https://", "http://"], ["", ""], $host);
 
-			if (isset($_POST['pumpio'])) {
-				PConfig::set(local_user(), 'pumpio', 'post'           , intval($_POST['pumpio']));
-				PConfig::set(local_user(), 'pumpio', 'import'         , $_POST['pumpio_import']);
-				PConfig::set(local_user(), 'pumpio', 'host'           , $host);
-				PConfig::set(local_user(), 'pumpio', 'user'           , $user);
-				PConfig::set(local_user(), 'pumpio', 'public'         , $_POST['pumpio_public']);
-				PConfig::set(local_user(), 'pumpio', 'mirror'         , $_POST['pumpio_mirror']);
-				PConfig::set(local_user(), 'pumpio', 'post_by_default', intval($_POST['pumpio_bydefault']));
-
-				if (!$_POST['pumpio_mirror']) {
-					PConfig::delete(local_user(), 'pumpio', 'lastdate');
-				}
+			PConfig::set(local_user(), 'pumpio', 'post'           , defaults($_POST, 'pumpio', false));
+			PConfig::set(local_user(), 'pumpio', 'import'         , defaults($_POST, 'pumpio_import', false));
+			PConfig::set(local_user(), 'pumpio', 'host'           , $host);
+			PConfig::set(local_user(), 'pumpio', 'user'           , $user);
+			PConfig::set(local_user(), 'pumpio', 'public'         , defaults($_POST, 'pumpio_public', false));
+			PConfig::set(local_user(), 'pumpio', 'mirror'         , defaults($_POST, 'pumpio_mirror', false));
+			PConfig::set(local_user(), 'pumpio', 'post_by_default', defaults($_POST, 'pumpio_bydefault', false));
+
+			if (!empty($_POST['pumpio_mirror'])) {
+				PConfig::delete(local_user(), 'pumpio', 'lastdate');
 			}
-			//header("Location: ".$a->get_baseurl()."/pumpio/connect");
```

Comment:
Please remove commented out code.

Reply:
I removed that code - that's why the line is red :-)

Reply:
*facepalm*